starknet_py/tests/e2e/docs/guide/test_using_cairo_serializer.py

```diff
@@ -64,7 +47,7 @@ async def test_using_cairo_serializer(network, gateway_account_client):
     await invoke_result.wait_for_acceptance()
 
     transaction_hash = invoke_result.hash
-    transaction_receipt = await client.get_transaction_receipt(transaction_hash)
+    transaction_receipt = await account_client.get_transaction_receipt(transaction_hash)
```

Comment:
Can we use `client` like it was previously?

Reply:
That's a tricky one because previously client was an AccountClient 😅 (check out a few previous lines)